setup.cfg

```
@@ -114,6 +114,10 @@ filterwarnings =
     ignore:PY_SSIZE_T_CLEAN will be required for '#' formats
     ignore:::astropy.tests.plugins.display
     ignore:::astropy.tests.disable_internet
+doctest_norecursedirs =
+    */setup_package.py
+doctest_subpackage_requires =
+    astropy/io/misc/asdf/*=asdf
```

Spaces around `=` is even more consistent! 😉 
```suggestion
    astropy/io/misc/asdf/* = asdf
```